### ISSUE TITLE
Incorrect behavior of Edit.OpenLineAbove in XML documentation comment

### ISSUE THREAD

**Issue by DustinCampbell** [dotnet/roslyn#2090]
**Repro Steps:**

* Add the following method to a class. Position the caret at the location of the `|`.
```C#
        /// <summary>
        /// stuff|
        /// </summary>
        public void Foo() { }
```
* Press Ctrl+Enter.

**Expected Result:**

The following text is the result, with the position of the caret indicated by `|`.
```C#
    /// <summary>
    /// |
    /// stuff
    /// </summary>
    public void Foo() { }
```

**Actual Result:**
```C#
    /// <summary>
    |
    /// stuff
    /// </summary>
    public void Foo() { }
```

**Comment by sharwell** [comment]
Sorry I forgot to move this. :no_mouth:

**Comment by Pilchie** [comment]
While this is irritating, I think it's going to be something we'll have to live with until 1.1

**Comment by DustinCampbell** [comment]
Fixed with #4813
